🇺🇦[Feature] added Ukrainian and Added more Unstopppable Domains extensions

kcw-grunt opened the pull request:
# Overview
Added Ukrainian language support due to the huge number of refugees unable to access their homes and protect themselves since [Feb 24, 2022](https://en.wikipedia.org/wiki/2022_Russian_invasion_of_Ukraine) added more access to Unstoppable Domains extensions to support more

kcw-grunt commented:
@mosadialiou just setting the pod version to v2.7.1 (5)

mosadialiou commented on loafwallet.xcodeproj/project.pbxproj:
```diff
@@ -5008,7 +5008,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = loafwallet/loafwallet.entitlements;
-				CURRENT_PROJECT_VERSION = 108;
+				CURRENT_PROJECT_VERSION = 5;
```
@kcw-grunt the downgrade 108->5 is safe?

kcw-grunt committed:
WIP: UD testing

kcw-grunt committed:
Merge branch 'develop' into feature/more-ud-extensions

kcw-grunt committed:
Adding український and added new UD extensions

- New UD key works
- Update the language support
- Added Ukrainian
- Removed Swedish, Danish, Dutch
- Relayout the UD Cell
- Added Ukrainian
- Enter a .crypto, .wallet, .zil, .nft, .blockchain, .bitcoin, .coin, .888, .dao, or .x domain.
- UD Placeholder
- SwiftUI ViewRepresentable from a UITextView
-Added український 🥳

kcw-grunt committed:
version bump

RyanPetter reviewed (approved):
Looks good!
